CHANGELOG.md

```diff
@@ -4,6 +4,112 @@ All notable changes to this project will be documented in this file.
+- The installer no longer creates a service-serving certificate
+  authority.  The certificate authority is now created by the
+  [service-CA operator][service-ca-operator].
+- On AWS, the worker IAM role permissions were reduced to a smaller
+  set required for kubelet initialization.
+- On AWS, the worker security group has been expanded to allow ports
+  9000-9999 for for host network services.  This matches the approach
+  we have been using for masters since 0.4.0.  The master security
+  group has also been adjusted to fix a 9990 -> 9999 typo from 0.4.0.
+- Several doc and internal cleanups and minor fixes.
+
+### Fixed
+
+- The router certificate authority is appended to the admin
+  `kubeconfig` to fix the OAuth flow behind `oc login`.
+- The installer now verifies cluster names supplied via
+  `install-config.yaml` (it previously only validated cluster names
+  provided via the install-config wizard).
+- Terraform variables are now generated from master machine
+  configurations instead of from the install configuration.  This
+  allows them to reflect changes made by editing master machine
+  configurations during [staged
+  installs](docs/user/overview.md#multiple-invocations).
+- `metadata.json` is generated before the Terraform invocation, fixing
+  a bug introduced in 0.12.0 which made it hard to clean up after
+  failed Terraform creation.
+- The machine-config server has moved its Ignition-config
+  service from port 49500 to 22623 to avoid the dynamic-port range
+  starting at [49152][rfc-6335-s6].
+- When the installer prompts for AWS credentials, it now respects
+  `AWS_PROFILE` and will update an existing credentials file instead
+  of erroring out.
+- On AWS, the installer now verifies that the user-supplied
+  credentials have sufficient permissions for creating a cluster.
+  Previously, permissions issues would surface as Terraform errors or
+  broken cluster functionality after a nominally successful install.
+- On AWS, the `destroy cluster` implementation is now more robust:
+    - The destroy code now checks for `nil` before dereferencing,
+      avoiding panics when removing internet gateways which had not
+      yet been associated with a VPC, and in other similar cases.
+    - The destoy code now treats already-deleted instances as
+      successfully deleted, instead of looping forever while trying to
+      delete them.
+- On OpenStack, the HAProxy configuration on the service VM now only
+  balances ports 80 and 443 across compute nodes (it used to also
+  balance them across control-plane nodes).
+- On OpenStack, the service VM now uses CoreDNS instead of dnsmasq.
+  And it now includes records for `*.apps.{cluster-domain}` and the
+  Kubernetes API.
+- On OpenStack, the service VM has been moved to its own subnet.
+
+### Removed
+
+- On AWS, control-plane nodes have been moved to private subnets and
+  no longer have public IPs.  Use a VPN or bastion host if you need
+  SSH access to them.
```

Comment:
@wking maybe you would consider to add the ssh pod solution which was mentioned in the PR?  Otherwise people will come back (natural i'd say) saying: _isn't installer's scope to give me a bastion too_ and not many will lookup up the conversation where it was mentioned the decision to not provide one

Reply:
> maybe you would consider to add the ssh pod solution...

You don't think they'd see [the docs][1]?

[1]: https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md#unable-to-ssh-into-master-nodes

Reply:
hopefully.  I guess is per individual in terms of the flow:

a) for each release they are looking at the change log notes _and_ scan the docs from A to Z to figure out if anything new was added/ removed OR
b) for each release they are looking only at the change log notes and doesn't review the docs (_assuming that nothing has changed).

is arguable imo but up to you